products/surveys/backend/max_tools.py

```diff
@@ -0,0 +1,247 @@
+                    "team_survey_config": self._get_team_survey_config(team),
+                }
+            )
+
+            # Convert to PostHog survey format and create directly
+            survey_data = self._convert_to_posthog_format(result, team)
+
+            # Use the proper serializer for validation and creation
+            from posthog.api.survey import SurveySerializerCreateUpdateOnly
+            from datetime import datetime
+
+            # Set launch date if requested
+            if result.should_launch:
+                survey_data["start_date"] = datetime.now()
+
+            # Create a minimal request-like object for the serializer context
+            class MinimalRequest:
+                def __init__(self, user):
+                    self.user = user
+                    self.method = "POST"
+
+            minimal_request = MinimalRequest(user)
+
+            serializer = SurveySerializerCreateUpdateOnly(
+                data=survey_data,
+                context={
+                    "request": minimal_request,
+                    "team_id": team.id,
+                    "project_id": team.project_id,
+                },
+            )
+
+            if serializer.is_valid() and len(survey_data["questions"]) > 0:
+                survey = serializer.save()
+                launch_msg = " and launched" if result.should_launch else ""
+
+                return f"✅ Survey '{survey.name}' created{launch_msg} successfully!", {
+                    "survey_id": str(survey.id),
+                    "survey_name": survey.name,
+                    "launched": result.should_launch,
+                    "questions_count": len(survey.questions),
+                }
+            else:
+                # Return validation errors
+                error_details = []
+                for field, errors in serializer.errors.items():
+                    if isinstance(errors, list):
+                        error_details.extend([f"{field}: {error}" for error in errors])
+                    else:
+                        error_details.append(f"{field}: {errors}")
+
+                return f"❌ Survey validation failed: {'; '.join(error_details)}", {
+                    "error": "validation_failed",
+                    "details": serializer.errors,
+                }
+
+        except Exception as e:
+            capture_exception(e, {"team_id": self._team_id, "user_id": user_id})
+            return f"❌ Failed to create survey: {str(e)}", {"error": str(e)}
+
+    def _get_team_survey_config(self, team: Team) -> dict[str, Any]:
+        """Get team survey configuration for context."""
+        survey_config = getattr(team, "survey_config", {}) or {}
+        return {
+            "appearance": survey_config.get("appearance", {}),
+            "default_settings": {"type": "popover", "enable_partial_responses": True},
+        }
+
+    def _get_existing_surveys_summary(self) -> str:
```

Comment:
You already have `existing_surveys` coming from the frontend, use one or the other?